code/game/objects/items/food/misc.dm

```diff
@@ -12,20 +20,12 @@
 	/// used to determine how much health rats/regal rats recover when they eat it.
 	var/rat_heal = 10
 
-/obj/item/food/cheese/wheel
-	name = "cheese wheel"
-	desc = "A big wheel of delcious Cheddar."
-	icon_state = "cheesewheel"
-	food_reagents = list(/datum/reagent/consumable/nutriment = 10, /datum/reagent/consumable/nutriment/protein = 5, /datum/reagent/consumable/nutriment/vitamin = 5) //Hard cheeses contain about 25% protein
-	w_class = WEIGHT_CLASS_NORMAL
-	rat_heal = 35
-
 /obj/item/food/cheese/wheel/Initialize(mapload)
```

Comment:
cheese/wheel isn't an item anymore, so this does nothing. If you changed it to just /obj/item/food/cheese it would let you cut cheese slices into more cheese so you are going to have to call /obj/item/food/cheese/wheel/MakeProcessable() on the cheese slice as well, but return early The same would go for any other subtypes of cheese. also, you will have to return early on Initialize mapload or else you will be able to fit items into cheese slices.